src/components/molecules/pokemonCard/PokemonCard.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from 'react';
+import Axios from 'axios';
+
+import { Card, Header, ImageContainer } from '../../atoms';
+import { Ability } from '../ability/Ability';
+
+import './PokemonCard.scss';
+
+export const PokemonCard = ({ url, name }) => {
+	const [avatar, setAvatar] = useState(null);
+	const [moves, setMoves] = useState(0);
+	const [weight, setWeight] = useState(0);
+	const [abilities, setAbilities] = useState(0);
+	const [isLoading, setIsLoading] = useState(false);
+
+	const cancelToken = Axios.CancelToken.source();
+
+	useEffect(() => {
+		const fetchCardData = async () => {
+			try {
+				setIsLoading(true);
+
+				const responseData = await Axios.get(`${url}`, {
+					cancelToken: cancelToken.token
+				}).catch((thrown) => {
```

Comment:
Hmmmm nu hebben we een dubbele catch, dat willen we niet! Volgens mij kan deze bij het `catch` blok worden bijgevoegd waarin je checkt of de error die wordt teruggegeven een `Axios.isCancel` is